packages/base/src/util/InvisibleMessage.js

```diff
@@ -0,0 +1,50 @@
+import InvisibleMessageMode from "../types/InvisibleMessageMode.js";
+import getSingletonElementInstance from "./getSingletonElementInstance.js";
+
+const styles = `position: absolute;
```

Comment:
I am not sure if the best way is to define styles and render the whole element globally... maybe we should try something like the createStyleInHead.js? Components/apps who need the InvisibleMessage will import it and then they will get the instance in the constructor of the component (create it) and the applications will get the instance in the earliest state? This will be the same as the approach in UI5. If you think that this is not needed at least wrap the style adding and the element construction in separate methods in order to make the code more beautiful and readable.

Reply:
After a discussion with @elenastoyanovaa and @MapTo0, we decided that we should stay with the current approach. In this way the util is instantiated when it's imported.